component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```
@@ -158,7 +160,7 @@ public Response createSpace(@Context UriInfo uriInfo,
                                                 "<br />\"visibility\": \"private\"," +
                                                 "<br />\"subscription\": \"validation\"<br />}" 
                                                 , required = true) SpaceEntity model) throws Exception {
-    if (model == null || model.getDisplayName() == null || model.getDisplayName().length() == 0) {
+    if (model == null || model.getDisplayName() == null || model.getDisplayName().length() == 0 || model.getDisplayName().length() > 200 || !pattern.matcher(model.getDisplayName()).matches()) {
```

IMO we should return an error 400 in this case, instead of a 401. It is a bad request (the request data are not correct to create a space), so it would make more sense to return a 400.